unittests/Feature/FeatureModelBuilder.cpp

```diff
@@ -34,81 +34,101 @@ TEST(FeatureModelBuilder, addOptionalFeature) {
   EXPECT_TRUE(FM->getFeature("a")->isOptional());
 }
 
-TEST(FeatureModelBuilder, addExclude) {
+// TODO(s9latimm): Replace string equals with subtree comparison
+TEST(FeatureModelBuilder, addExcludeConstraint) {
   FeatureModelBuilder B;
-
   B.makeFeature<BinaryFeature>("a");
-  B.addExclude("a", "b")->makeFeature<BinaryFeature>("b");
+  B.makeFeature<BinaryFeature>("b");
+  auto C = std::make_unique<ImpliesConstraint>(
+      std::make_unique<PrimaryFeatureConstraint>(
+          std::make_unique<Feature>("a")),
+      std::make_unique<NotConstraint>(
+          std::make_unique<PrimaryFeatureConstraint>(
+              std::make_unique<Feature>("b"))));
```

Comment:
Seeing this multiple times, maybe it would make sense to create a ConstraintBuild but we can also do this later in a refactoring. If you want to do it later, could you please create an issue for it :smile: